### PR TITLE
Feat: Product Listing Page

### PR DESCRIPTION
**Pull request by AmishMishra11**
Added Product Listing page By using Mockbee backend,
The functionality will be added later

https://user-images.githubusercontent.com/76205249/158974341-cb7c6c10-3e20-441e-8da1-01c90bd360a3.mp4

### CODE REVIEW

File: src/Components/Card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./styles.css";
+const Card = ({ item }) => {
+  const { title, price, rating, imageURL } = item;
```
Comment (mnath30) [inline src/Components/Card/index.jsx:4]: Liked the way how you destructured to avoid repetition

File: src/Components/Card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <div className="card">
+      <img className="item" src={imageURL} />
```
Comment (mnath30) [inline src/Components/Card/index.jsx:7]: You can also add alt to images

File: src/Components/Card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <h2>{title}</h2>
+      <h4 className="flex-r rating">
+        <div>₹{price}</div>
+        <div>
+          {rating}
+          <i className="fas fa-star star"></i>
+        </div>
+      </h4>
+      <div className="card-buy flex-r">
+        <a className="card-btn btn-primary border-radius-S" href="">
```
Comment (mnath30) [inline src/Components/Card/index.jsx:20]: Good use of utility classes

File: src/Components/Card/styles.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.btn-primary {
+  color: white;
```
Comment (mnath30) [inline src/Components/Card/styles.css:2]: You could use variable names here instead of directly using the colors

**Review by mnath30 — APPROVED** [review]
Code is clean 
Good use of utility classes and destructuring of objects

File: src/Components/Card/styles.css (hunk 4 above)
Comment (prathmeshjagtap) [inline src/Components/Card/styles.css:2]: you can use variable naming here

**Review by prathmeshjagtap — APPROVED** [review]
code is Well structured 
I like that you  have include input tag in label

File: src/Components/Card/index.jsx (hunk 2 above)
Comment (AmishMishra11) [reply src/Components/Card/index.jsx:7]: yes i will do that in the next pr thank you

File: src/Components/Card/styles.css (hunk 4 above)
Comment (AmishMishra11) [reply src/Components/Card/styles.css:2]: Only for black and white colors I use this otherwise I use variable naming.

Comment (AmishMishra11) [reply src/Components/Card/styles.css:2]: Only for black and white colors, I use this otherwise I use variable naming

**Pull request merged**